test/run_tests_tls_netaccess

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+
+
+function test_tls_server {
+  msg=$1
+  suite=$2
+  version=$3
+  EXIT_CODE=test_tls_server_tmp_code
+  OSSL_STDO=test_tls_server_tmp_openssl_stdout
+  OSSL_STDE=test_tls_server_tmp_openssl_stderr
+  SCAPY_STDE=test_tls_server_tmp_scapy_stderr
+
+  echo "1" > $EXIT_CODE
+  ($TRAVIS_SUDO python tls_test_server.py "$msg" 2>$SCAPY_STDE; echo $? > $EXIT_CODE) &
```

Comment:
Have you tried to replace this by something like used here:
https://github.com/secdev/scapy/blob/0c9b9089abffcffc788cc525ac7f2905c840d49d/appveyor.yml#L27
To make sure that codecoverage is used ?